### ISSUE TITLE
FFT of 1 element leading estimates negative results

### ISSUE THREAD

**Issue by mmaker** [mmaker/zkalc#34]
![signal-2023-08-12-163653](https://github.com/mmaker/zkalc/assets/278405/7b2c705d-db35-4dc4-a1e4-f04ca5899a2d)

**Comment by mmaker** [comment]
The function for estimating fft's is here: https://github.com/mmaker/zkalc/blob/main/frontend/lib/estimates.js#L153

**Comment by asn-d6** [comment]
Yep, there is something rotten with FFTs on zkalc. Even a 2^20 FFT over curve25519 will return a negative value.

**Comment by mmaker** [comment]
if you don't have examples to extrapolate from, what we do is we take the last few samples and do regression from there. Do you have a better idea to go about it, or am I misunderstanding the question?
